assets/i18n/en.json

```diff
@@ -798,6 +967,14 @@
     "I18N_TOPIC_VIEWER_STUDY_SKILLS_SUBTITLE": "Use the following Review Cards to help you study skills about <[topicName]>.",
     "I18N_TOPIC_VIEWER_VIEW_ALL": "View All",
     "I18N_TOPIC_VIEWER_VIEW_LESS": "View Less",
+    "I18N_TOPIC_dLmjjMDbCcrf_DESCRIPTION": "You'll often need to solve problems with unknown numbers -- for example, if you've bought an item that is on sale and want to find out the original price. In this topic, you'll learn how to do this with equations, expressions, and formulae.",
+    "I18N_TOPIC_dLmjjMDbCcrf_TITLE": "Expressions and Equations",
+    "I18N_TOPIC_iX9kYCjnouWN_DESCRIPTION": "Did you know that all possible numbers of things can be expressed using just ten digits (0,1,2,3,...,9)? In this topic, we'll learn how we can use place values to do that, and, for example, how '5' has a different value in '25' and '2506'.",
```

Comment:
Just FYI, I updated this on the website since the current wording is a little confusing. Here's the new text:

`Did you know that all possible numbers of things can be expressed using just ten digits (0,1,2,3,...,9)? In this topic, we'll learn how we can use place values to do that, and see why "5" has a different value in "25" and "2506".`

Reply:
Done!